src/command-registry.coffee

```diff
@@ -154,16 +157,14 @@ class CommandRegistry
       for name, listeners of @inlineListenersByCommandName
         if listeners.has(currentTarget) and not commandNames.has(name)
           commandNames.add(name)
-          commands.push({name, displayName: _.humanizeEventName(name)})
+          commands.push(commandFromListener(name, listeners.get(currentTarget)))
```

Comment:
Could we do this logic at registration time instead? We can make the new object based format the canonical representation of a command and just synthesize the displayName if a function is registered.